engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -317,9 +308,9 @@ private void generateSkyBand(FBO skyBand) {
         }
 
         skyBand.bind();
-        graphicState.setRenderBufferMask(skyBand, true, false, false);
+        setRenderBufferMask(skyBand, true, false, false);
 
-        setViewportTo(skyBand.dimensions());
+        setViewportToSizeOf(skyBand.dimensions());
```

Comment:
Such as `public void generateSkyBands()` method in `PostProcessor`. But all of these will be removed once all nodes are in place. Best to begin the huge refactor then.